src/components/pages/Thanks.tsx

```diff
@@ -28,6 +30,36 @@ export const Thanks: React.FC = () => {
     <>
       <br />
       <Container>{JSON.stringify(data)}</Container>
+      <Card style={{ width: '18rem' }}>
```

Comment:
Move the ThanksCard Component into it's own Folder in the UI folder 
this will make it easier to declare Prop types, Styles and make it 
cleaner to work with.